mainnet-1/config/pre-genesis.json

```diff
@@ -3969,7 +3969,7 @@
             "denom": "utgd",
             "amount": "1000000"
```

Comment:
This has to be increased as well. Twice the amount here will be distributed between the arbiters when rendering the decision.